wiki/FAQ/en.md

```diff
@@ -296,23 +291,22 @@ If you have any questions regarding pp, please refer to the [wiki page for pp fi
 #### Where can I find and get the beatmaps?
 _Detailed explanation of the beatmaps: [Beatmaps][Beatmaps wikilink]_
 
-You can find it from the main page under the name "[Beatmaps][Beatmaplist link]".
--   Just remember to log in to your osu! account before-hand or the server treats you as a guest and prohibit you from downloading beatmaps from the site.
+You can find it from the main page under the name "[Beatmaps][Beatmaplist link]". Just remember to log in to your osu! account beforehand or the server treats you as a guest and prohibit you from downloading beatmaps from the site.
 
 #### Can I disable "Kiai Time"?
-No, because unlike video, they are integrated to the game (as a part with the hit objects), thus cannot be disabled.
+**No**, because unlike video, it was integrated to the game (as a part with the hit objects), thus cannot be disabled.
 
 #### The maps are too hard! Where can I find some easier maps?
 Pokebis has compiled a set of beatmaps suitable for beginners to play. You can find them here: <https://osu.ppy.sh/forum/t/5456>
```

Comment:
Not everyone knows who Pokebis is, I recommend changing it to `Some members have compiled a beatmap set that is suitable for beginners. You can find them here: <https://osu.ppy.sh/forum/t/5456>` and updating the link to a more recent set if it's there.